### PR TITLE
feat:add UI screen

### PR DESCRIPTION
**Pull request by rahulmistry751**
Added home, explore, bookmark UI
Added basic route

### CODE REVIEW

**Review by ScoobyDubeyDo — APPROVED** [review]
- Nice UI
- I liked the module CSS approach

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,24 @@
...
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
+    <div className="App border-box font-fira bg-background">
+      <div className="container mx-auto px-4">
+        <Navbar/>
+        <div className="cols">
+          {/* <div className="cols-3"> */}
```
Comment (Pratik1005) [inline src/App.js]: remove log if not needed

File: src/App.js [hunk 2]
```diff
@@ -1,41 +1,24 @@
...
+          <Sidebar/>
+          <Routes>
+            <Route path="/" element={<Home/>}></Route>
+            <Route path="/bookmarks" element={<Bookmarks/>}></Route>
+            <Route path="/explore" element={<Explore/>}></Route>
+          </Routes>
+          <Suggestions/>
+          {/* </div> */}
```
Comment (Pratik1005) [inline src/App.js]: same here

**Review by Pratik1005 — APPROVED** [review]
Clean UI, good to merge

File: src/features/Home/Home.module.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+/* Single post */
+.singlepost-container{
+    display: flex;
+    flex-wrap: wrap;
+}
+.post-option{
+    margin-left:auto;
+}
+.post-content{
+
```
Comment (simran61) [inline src/features/Home/Home.module.css:36]: can remove this class if it has no styles in it

File: src/index.css [hunk 4]
```diff
@@ -1,13 +1,31 @@
...
+  --primary:#2563EB;
+  --primary-accent:#3B82F6;
+}
+.wrapper{
+
+  margin:auto;
+}
+.container{
+  min-height:100vh;
+  min-width: 769px;
```
Comment (simran61) [inline src/index.css]: can use relative units instead of px

File: src/index.css [hunk 5]
```diff
@@ -1,13 +1,31 @@
...
+}
+.cols{
+  display: grid;
+  grid-template-columns: 1fr 3fr 2fr;
+}
+/*Media query */
+@media (min-width: 1536px){
+  .container{
+    width:1280px;
```
Comment (simran61) [inline src/index.css]: can use relative units here as well

**Review by simran61 — APPROVED** [review]
Looks good

**Pull request merged**